root/static/scripts/edit/URLCleanup.js

```diff
@@ -1438,10 +1458,16 @@ const CLEANUPS = {
       }
       if (/^https:\/\/www\.instagram\.com\/explore\//.test(url)) {
         return {
-          error: l(
-            `Explore links are not officially connected to a location.
-             If you want to link a to a location, try and find
-             the place's Instagram profile instead, if there's one.`,
+          error: exp.l(
+            `Instagram “{explore_url_pattern}” links are not officially
+             connected to a location.
+             If you want to link a place to a location, try and find
+             the place’s Instagram profile instead, if there’s one.`,
```

Comment:
While you're changing the string I'd spell out "is" here because the stress is on that word, not "one"
```suggestion
             the place’s Instagram profile instead, if there is one.`,
```

Reply:
Fixed in https://github.com/metabrainz/musicbrainz-server/pull/1536/commits/9531a73781dbb17b64e7bb0f28846070fdccb0a1